src/components/Pickers/BasePicker.Props.ts

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { IPickerItemProps } from './PickerItem.Props';
+
+// Type T is the type of the item that is displayed
+// and searched for by the people picker. For example, if the picker is
+// displaying persona's than type T could either be of Persona or Ipersona props
+export interface IBasePickerProps<T> extends React.Props<any> {
+  /**
+   * Function that specifies how the selected item will appear.
+   */
+  onRenderItem?: (props: IPickerItemProps<T>) => JSX.Element;
+  /**
+   * Function that specifies how an individual suggestion item will appear.
+   */
+  onRenderSuggestionsItem?: (props: T) => JSX.Element;
```

Comment:
i kind of wish this was onRenderSuggestions: ISuggestionsProps<T> rather than "Item"